util/Conventional_Monitor/image_gen/sorc/conmon_read_pw_IG.fd/CMakeLists.txt

```diff
@@ -5,7 +5,7 @@ cmake_minimum_required(VERSION 2.6)
   add_executable(conmon_read_pw_IG.x  ${CONMON_READ_PW_IG_SRC} )
   set_target_properties( conmon_read_pw_IG.x PROPERTIES COMPILE_FLAGS ${CONMON_READ_PW_IG_Fortran_FLAGS} )
   include_directories( ${CORE_INCS} ${NCDIAG_INCS} ) 
-  target_link_libraries( conmon_read_pw_IG.x ${W3NCO_4_LIBRARY} ${NCDIAG_LIBRARIES} ${NETCDF_LIBRARIES_F90} ${NETCDF_LIBRARIES} ${HDF5_Fortran_HL_LIBRARIES} ${HDF5_LIBRARIES} ${MPI_Fortran_LIBRARIES} )
+  target_link_libraries( conmon_read_pw_IG.x ${W3NCO_4_LIBRARY} ${NCDIAG_LIBRARIES} ${NETCDF_LIBRARIES_F90} ${NETCDF_LIBRARIES} ${MPI_Fortran_LIBRARIES} )
```

Comment:
and here.

Reply:
Update to use `${NETCDF_Fortran_LIBRARIES} ${NETCDF_C_LIBRARIES}` in `CMakeLists.txt` at source code level.